config/modules/curate.cfg

```diff
@@ -33,92 +33,23 @@ plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.METSRes
 
 # CHECKM TASK SETTINGS:
 # If you would like to use the Checkm Manifest validation, you can also
 # add the following five tasks to the above list:
-#    org.dspace.ctask.replicate.checkm.TransmitManifest = transmitmanifest, \
-#    org.dspace.ctask.replicate.checkm.VerifyManifest = verifymanifest, \
-#    org.dspace.ctask.replicate.checkm.FetchManifest = fetchmanifest, \
-#    org.dspace.ctask.replicate.checkm.CompareWithManifest = auditmanifest, \
-#    org.dspace.ctask.replicate.checkm.RemoveManifest = removemanifest
+# plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.checkm.TransmitManifest = transmitmanifest
+# plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.checkm.VerifyManifest = verifymanifest
+# plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.checkm.FetchManifest = fetchmanifest
+# plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.checkm.CompareWithManifest = auditmanifest
+# plugin.named.org.dspace.curate.CurationTask = org.dspace.ctask.replicate.checkm.RemoveManifest = removemanifest
 
 ## task queue implementation
 plugin.single.org.dspace.curate.TaskQueue = org.dspace.curate.FileTaskQueue
-## For better Replication Task Suite performance, you may wish to enable the below
-## FilteredFileTaskQueue instead (it attempts to filter duplicate tasks out of the task queue)
-# plugin.single.org.dspace.curate.TaskQueue = org.dspace.ctask.replicate.FilteredFileTaskQueue
 
 # directory location of curation task queues
 curate.taskqueue.dir = ${dspace.dir}/ctqueues
 
 # (optional) directory location of scripted (non-java) tasks
 # curate.script.dir = ${dspace.dir}/ctscripts
-
-# Friendly names for curation tasks to appear in admin UI
-# Also acts as a filter - i.e. tasks not enumerated here can still
-# be invoked on cmd line, etc - just not in UI
-curate.ui.tasknames = profileformats = Profile Bitstream Formats
-curate.ui.tasknames = requiredmetadata = Check for Required Metadata
-curate.ui.tasknames = checklinks = Check Links in Metadata
-curate.ui.tasknames = estaipsize = Estimate Storage Space for AIP(s)
-curate.ui.tasknames = readodometer = Read Odometer
-curate.ui.tasknames = transmitaip = Transmit AIP(s) to Storage
-curate.ui.tasknames = verifyaip = Verify AIP(s) exist in Storage
-curate.ui.tasknames = fetchaip = Fetch AIP(s) from Storage
-curate.ui.tasknames = auditaip = Audit against AIP(s)
-curate.ui.tasknames = removeaip = Remove AIP(s) from Storage
-curate.ui.tasknames = restorefromaip = Restore Missing Object(s) from AIP(s)
-curate.ui.tasknames = replacewithaip = Replace Existing Object(s) with AIP(s)
-curate.ui.tasknames = restorekeepexisting = Restore Missing Object(s) but Keep Existing Objects
-curate.ui.tasknames = restoresinglefromaip = Restore Single Object from AIP
-curate.ui.tasknames = replacesinglewithaip = Replace Single Object with AIP
```

Comment:
I'm assuming that these labels were moved elsewhere for DSpace 7 (maybe messages.xml?) If they don't exist anywhere, that could be an issue.

Reply:
These labels have generally been migrated to Angular, here:

https://github.com/DSpace/dspace-angular/blob/main/src/assets/i18n/en.json5#L1113-L1123

My understanding of how Angular now shows these is, it queries the server configuration via REST, for the list of possible curation task ids via this component, which lives at the /admin/curation-tasks route:

typescript: https://github.com/DSpace/dspace-angular/blob/main/src/app/curation-form/curation-form.component.ts#L52-L61
html: https://github.com/DSpace/dspace-angular/blob/main/src/app/curation-form/curation-form.component.html#L8

Absent configuration in src/assets/i18n/en.json5, the RTS tasks would show in the DSpace 7 admin "Curation Tasks" page dropdown, simply with their string id rather than their friendly name.

For RTS, I guess this could all be covered in the install documentation, explaining that these would need to be added to the appropriate i18n file(s) in Angular in order to have human-readable names.

Reply:
Yes, it sounds like the resolution here is to update the RTS documentation to indicate that these labels need to be added to en.json5 (and other languages as needed) so that they are displayed properly by the Angular UI. This label update will need to be tested to verify that it works as expected.

The question then is whether to hold up this PR for that documentation update to be made or just capture the full list of labels in a new documentation ticket and allow this PR to proceed. As noted in the PR description, there will also need to be a documentation change to update the dependencies to be added to the pom.